README.md

```diff
@@ -26,6 +28,35 @@ Install the plugin by adding this code to your lazy.vim config:
 }
 ```
 
+### Using mini.deps
+
+```lua
+local MiniDeps = require('mini.deps')
+MiniDeps.add({
+  source = 'sourcegraph/amp.nvim',
+})
+
+require('amp').setup({ auto_start = true, log_level = "info" })
```

Comment:
should we perhaps recommend using [`MiniDeps.now`](https://github.com/nvim-mini/mini.deps?tab=readme-ov-file#lazy-loading) e.g, `MiniDeps.now(function() require('amp').setup({ auto_start = true, log_level = "info" }) end)`?